docs/websites/cms/install-and-configure-wordpress-using-wp-cli.md

```diff
@@ -0,0 +1,273 @@
+
+## Basics of WP-CLI
+Before we get into the helm of the things, let us learn some essential basics of how exactly WP-CLI works. This would help you feel comfortable with what is going to follow next.
+
+So far we have seen WP-CLI is accessed through the keyword `wp`. What follows that is a set of commands and their own sub commands. For example we have a command to download WordPress which goes like `wp core download`. Here wp is the main command while core and download are its sub-commands. There can be several sub-commands for each sub-command as well.
+
+WP-CLI also comes with a pretty detailed help section which will allow you to keep tab on all the commands you would need. You can access it by typing
+
+    wp help
+
+Do that and you will get a screen similar to
+
+    wp
+
+    DESCRIPTION
+
+    Manage WordPress through the command-line.
+
+    SYNOPSIS
+
+    wp <command>
+
+    SUBCOMMANDS
+
+    cache               Manage the object cache.
+    cap                 Manage user capabilities.
+    cli                 Get information about WP-CLI itself.
+    comment             Manage comments.
+    core                Download, install, update and otherwise manage WordPress proper.
+    cron                Manage WP-Cron events and schedules.
+    db                  Perform basic database operations.
+    eval                Execute arbitrary PHP code after loading WordPress.
+    eval-file           Load and execute a PHP file after loading WordPress.
+    :
+
+
+`:` is a prompt where you need to enter few commands to navigate through this help menu. Up and down arrow keys will take you through the complete help. And pressing q will exit the help menu. That's all you need to know for now. For complete detail on how to navigate through the complete help section, you can always type `h` at the above prompt.
+
+In our last step, we enabled Bash Completion feature for WP-CLI. To use that type `wp` and press tab twice. You would see the list of available commands. You can repeat the same by typing `wp core` and then pressing tab twice. Now you will see a list of commands that can be used with `core`.
+
+## Installing WordPress
+
+### Setting up Database
+
+1.  Before you proceed, you need to setup a database first. For that, login to the MySQL server first. Replace `user` with your MySQL user.
+
+        mysql -u user -p
+
+2.  Next step is to create a database.
+
+        create database wordpress;
+
+3.  Now we need to grant all the required privileges for the database to the mysql user.
+
+        grant all on wordpress.* to 'user' identified by 'password';
+
+4.  Type `quit` to exit the MySQL command line.
+
+### Main Install
+
+1.  Move to the user directory where you would be installing the blog. Traditionally it should be in `/var/www/html` directory. Create a new user directory for your user if it doesn't exist already. Also proper group and ownership permissions need to be given to the user.
```

Comment:
This section is confusing because it references both a user directory and `/var/www/html` as the document root. Because you have specified our LAMP guide as a prerequisite, you can safely assume that the reader is using `/var/www/html` and that the user is www-data.